cmake/autocoder/ai_xml.cmake

```diff
@@ -76,7 +76,8 @@ macro(__ai_info XML_PATH MODULE_NAME)
         # Next parse the output matching one line at a time
         read_from_lines("${AI_OUTPUT}" XML_TYPE_FROM_FILE MODULE_DEPENDENCIES FILE_DEPENDENCIES)
     endif()
-    list(APPEND FILE_DEPENDENCIES "${FPRIME_AC_CONSTANTS_FILE}")
+    # Post v3.4.1, support for the AcConstants.ini file has been dropped in favor of AcConstants.fpp,
+    # list(APPEND FILE_DEPENDENCIES "${FPRIME_AC_CONSTANTS_FILE}")
```

Comment:
The Ai.xml is still used for dictionary generation.  By dropping this here, do we not end up in  a situation where the `Ai.xml` is wrong?

Reply:
In which case would dropping this corrupt the dictionary ? 
Agreed that a more conservative approach could be to wait for the JSON dictionary to come in and drop the entire XML/Autocoder stuff at once... I went for it because we planned this ticket for this sprint haha

Reply:
Delete!